package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "4.2.0",
+  "version": "4.2.1",
```

Comment:
This project is [already at v4.2.1](https://github.com/bcgov/business-create-ui/blob/c1afbed036ccd93f75b7da4be7994e3ff79c2647/package.json#L3). Please rebase and then update app version again.